.github/workflows/merge.yml

```
@@ -0,0 +1,43 @@
+name: merge
+
+on:
+  push:
+    branches: [ main ]
+
+  workflow_dispatch:
+
+concurrency: prr:deploy
+
+jobs:
+  merge:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [16.x]
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        ref: 'next'
+        fetch-depth: 0
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v3
+      with:
+        node-version: ${{ matrix.node-version }}
+        cache: 'npm'
+    - run: npm ci
+    - run: npm run build --if-present
```

Flag shouldn't be necessary, and I'd rather error if the build system somehow gets unplugged.
```suggestion
    - run: npm run build
```